### DISCUSSION TITLE
Receiving PO line items using API

### DISCUSSION THREAD

**Discussion by Chanzudl** [inventree/InvenTree#5478] [Q&A] [answered]
How can I receive line items in a Purchase Order through the API?

**Comment by miggland** [comment, marked as answer]
The endpoint is
`api/order/po/{id}/receive/`

Check the API docs for details:
https://demo.inventree.org/api-doc/#tag/order/operation/order_po_receive_create

The python binding has a method to do this, see the function description.

**Reply by Chanzudl** [reply]
Thank you. Worked like a charm.
